tests/test_backend.py

```diff
@@ -9,6 +10,8 @@ def test_create_user_adds_multiple_elements(self):
         self.assertEquals(len(sut._user_list), 1)
         sut.create_user("andrew@gmail.com")
         self.assertEquals(len(sut._user_list), 2)
+        sut.create_user("can@gmail.com")
+        self.assertEquals(len(sut._user_list), 3)
```

Comment:
What is this assertion testing? I don't see it being different from the two assertions above.